db/migrate/20150513034751_add_repository_branches.rb

```diff
@@ -0,0 +1,26 @@
+class AddRepositoryBranches < ActiveRecord::Migration
+  def up
+    create_table :katello_repository_branches do |t|
+      t.string :branch, :null => false
```

Comment:
Based on that example I am inclined to agree with 'ref' for the property name, +1